<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>464</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="principal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-50</y>
      <width>801</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbAki">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>110</y>
       <width>241</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="akinatorimg/imgs.qrc">:/aki_expressions/aki01.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>361</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);
border: 1px solid #000;
border-radius: 40px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 16pt &quot;Segoe UI&quot;;
border: none;</string>
      </property>
      <property name="text">
       <string>I THINK OF</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbImg">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 0;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 16pt &quot;Segoe UI&quot;;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbSubName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 10pt &quot;Segoe UI&quot;;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnPlayAgn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>430</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  	align-items: center;
  	background-color: #4849EF;
	border: 1px solid #212151;
  	border-radius: 6px;
	box-sizing: border-box;
 	color: rgb(255, 255, 255);
	cursor: pointer;
	font-family: 'Arial', sans-serif;
	font-weight: 700;
	font-size: 15px;
	cursor: pointer;
}

QPushButton:hover {
	transition: 1s;
	background-color:  #181847;
}</string>
     </property>
     <property name="text">
      <string>Play Again</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>130</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);
border: 1px solid #000;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>170</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);
border: 1px solid #000;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbRoom">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>1001</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>QPushButton {
  align-items: center;
  background-color: rgb(238, 139, 0);
  border-radius: 6px;
  box-shadow: transparent 0 0 0 3px,rgba(18, 18, 18, .1) 0 6px 20px;
  box-sizing: border-box;
  color: rgb(255, 255, 255);
  cursor: pointer;
  display: inline-flex;
  flex: 1 1 auto;
  font-family: Inter,sans-serif;
  font-size: 1.2rem;
  font-weight: 700;
  justify-content: center;
  line-height: 1;
  margin: 0;
  outline: none;
  padding: 1rem 1.2rem;
  text-align: center;
  text-decoration: none;
  transition: box-shadow .2s,-webkit-box-shadow .2s;
  white-space: nowrap;
  border: 0;
  user-select: none;
  touch-action: manipulation;
}

QPushButton:hover {
	box-shadow: #121212 0 0 0 3px, transparent 0 0 0 0;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>room_akinator.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="principal_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>480</y>
       <width>801</width>
       <height>651</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>361</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);
border: 1px solid #000;
border-radius: 40px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 600 16pt &quot;Segoe UI&quot;;
border: none;</string>
       </property>
       <property name="text">
        <string>I THINK OF</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbImg_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>191</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 0;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lbName_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>250</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 600 16pt &quot;Segoe UI&quot;;
border: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbSubName_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>280</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 400 10pt &quot;Segoe UI&quot;;
border: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnBack_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>430</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  	align-items: center;
  	background-color: #4849EF;
	border: 1px solid #212151;
  	border-radius: 6px;
	box-sizing: border-box;
 	color: rgb(255, 255, 255);
	cursor: pointer;
	font-family: 'Arial', sans-serif;
	font-weight: 700;
	font-size: 15px;
	cursor: pointer;
}

QPushButton:hover {
	transition: 1s;
	background-color:  #181847;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>130</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);
border: 1px solid #000;
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255);
border: 1px solid #000;
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <zorder>lbRoom</zorder>
    <zorder>lbAki</zorder>
    <zorder>frame</zorder>
    <zorder>btnPlayAgn</zorder>
    <zorder>frame_2</zorder>
    <zorder>frame_3</zorder>
    <zorder>principal_2</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="akinatorimg/imgs.qrc"/>
 </resources>
 <connections/>
</ui>
